From a41e60c5d722275dbb3d20aa023b2fc3e629262e Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 22 Nov 2018 13:20:26 -0800
Subject: [PATCH] MANTA-4030 many images using node 0.10 or 0.12 broken by
 transitive lru-cache@4.1.4 dep Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
 Approved by: Josh Wilsdon <josh@wilsdon.ca>

---
 package.json | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/package.json b/package.json
index 6c1569d..465a879 100644
--- a/package.json
+++ b/package.json
@@ -1,15 +1,16 @@
 {
     "name": "mahi",
     "description": "Mahi client",
-    "version": "2.2.0",
+    "version": "2.2.1",
     "author": "Joyent (joyent.com)",
     "main": "index.js",
+    "// lru-cache": "attempting to pin version while satisfying transitive deps, because lru-cache@4.1.4 includes yallist@3 which breaks node --abort-on-uncaught-exception with node <v4",
     "dependencies": {
         "aperture": "git://github.com/joyent/node-aperture.git#master",
         "assert-plus": "0.1.5",
         "bunyan": "1.8.1",
         "http-signature": "0.10.0",
-        "lru-cache": "2.5.0",
+        "lru-cache": "4.1.3",
         "restify": "4.1.1",
         "verror": "1.3.7"
     },
-- 
2.21.0

